### PR TITLE
First update

### PR DESCRIPTION
**Pull request by rfreichel**
I added functions and mainly worked on the tests for the R package.

### CODE REVIEW

**Comment by fdabl** [comment]
@mrooyakkers and @claravetter please review this pull request if you have time! Since this is a day late, you also have a day longer. However, if you can't make it, don't worry.

File: eegstats/R/main.R [hunk 1]
```diff
@@ -1,69 +1,299 @@
...
+# 5. grand average ersp
+#' grand_average_ersp
+#' @description  The function \emph{grand_average_ersp} computes the grand average  ERSP signal for all conditions and all frequencies from one electrode.
+#' @param my_erspdata Input Matrix with shape: subject ID x condition x timex frequency (from only one electrode).
+#' @details For more details, see Wikipedie page: \href{https://sccn.ucsd.edu/wiki/Chapter_11:_Time/Frequency_decomposition}{article}
+#' @return A matrix that contains the grand average ERSP for all conditions and all frequencies of one electrode. The matrix will be of shape condition x time x frequency.
+#' @export grand_average_ersp_data
+#' @seealso Add example of similar functions [still to be done].
+
+grand_average_ersp <- function(my_erspdata) {
```
Comment (claravetter) [inline eegstats/R/main.R:209]: according to the style guide, function names should be verbs, so maybe change it to something like compute_grand_average_ersp. Even though that is very long but maybe you can come up with something better ;)

File: eegstats/R/main.R [hunk 2]
```diff
@@ -1,69 +1,299 @@
...
+#' @param ncond A numeric value that specifies which condition should be plotted (e.g. 1 for condition 1)
+#' @return A time - frequency plot with the grand average ERSP signal.
+#' @seealso Add example of similar functions [still to be done].
+
+# I wanted to include an xample output figure in the documentation:
+#\if{html}{\figure{erp.png}{options: width=100 alt="erp.png"}}
+#\if{latex}{\figure{erp.png}{options: width=7cm}}
+# This didn't work - any suggestions how to add images to R function documentation?
+
+grand_average_ersp_plot <- function(my_erspdata, grand_average_ersp_data, n_cond) {
```
Comment (claravetter) [inline eegstats/R/main.R:243]: same here

File: eegstats/R/main.R [hunk 3]
```diff
@@ -1,69 +1,299 @@
...
+
+  for (i in 1:length(desired_timepoints)) {
+
+    j <- desired_timepoints[i]
+
+    index_time[i] <- which(time == j)
+
+  }
+
+  labRow_label <- NA
```
Comment (claravetter) [inline eegstats/R/main.R:264]: Object names are not supposed to have upper case letters (style guide)

**Pull request closed without merging**